src/test/kotlin/dev/panuszewski/gradle/IncludedBuildSpec.kt

```diff
@@ -102,4 +102,19 @@ class IncludedBuildSpec : GradleSpec() {
         // then
         result.buildOutcome shouldBe BUILD_SUCCESSFUL
     }
+
+    @ParameterizedTest
+    @SupportedIncludedBuilds
```

Comment:
Since we always use `build-logic` in the `CyclicBuildHierarchy` fixture, we don't have to run parameterized test for all kinds of included builds (we don't use the parameter anyway)
```suggestion
    @Test
```